.github/workflows/CI-MATs.yml

```
@@ -80,31 +106,25 @@
+          path: ${{ github.workspace }}
       - name: Run Tests for python ${{ matrix.python-version }}
         id: tests
-        run: make -j1 -f Makefile test ;
+        run: make -j1 -f Makefile test-mats ;
       - name: Summerize MATs for python ${{ matrix.python-version }}
         id: sumerize-mats
         run: |
           echo "- [x] MATS succeeded with python version ${{ matrix.python-version }}" >> "$GITHUB_STEP_SUMMARY"
         if: ${{ success() }}
+      - name: Summerize MATs for python ${{ matrix.python-version }} (FAILED)
+        id: sumerize-mats
```

actionlint (1.7.4)

step ID "sumerize-mats" duplicates. previously defined at line:119,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)